src/content/docs/en/guides/assets.mdx

```diff
@@ -0,0 +1,277 @@
+
+<!-- Remote image on another server -->
+![Astro logo](https://astro.build/assets/logo.png)
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro logo">
+```
+
+### Convert from `@astrojs/image`
+
+**Built-in asset support is incompatible with the `@astrojs/image` integration.**
+
+To avoid errors in your project, complete the following steps:
+
+1. Remove the `@astrojs/image` integration.
+
+    You must [remove the integration](/en/guides/integrations-guide/#removing-an-integration) by uninstalling and then removing it from your `astro.config.mjs` file.
+
+2. Migrate any existing `<Image />` components.
+
+    Change all `import` statements from `@astrojs/image/components` to `astro:assets` to use the new built-in `<Image />` component.
+
+    Remove any component attributes that are not [currently supported image asset properties](#properties).
+    
+    For example `aspectRatio` is no longer supported, as it is now automatically inferred from the `width` and `height` attributes.
+
+      ```astro title="src/components/MyComponent.astro" del= {2,11} ins={3}
+      ---
+      import { Image } from '@astrojs/image/components';
+      import { Image } from 'astro:assets'
+      import localImage from "../assets/logo.png";
+      const localAlt = "The Astro Logo";
+      ---
+
+      <Image
+        src={localImage}
+        width={300}
+        aspectRatio="16:9"
+        alt={localAlt}
+      />
+      ```
+
+
+3. Remove any existing `<Picture />` components
+
+    Currently, the built-in assets feature does not include a `<Picture />` component.
+
+    Instead, you can use the HTML image attributes `srcset` and `sizes` or the `<picture>` tag [for art direction or to create responsive images](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction).
+
+
+### Update content collections schemas
+
+A new `image` helper for content collections can validate the metadata of images located in the `src/assets/` folder, such as author photos or blog cover images.
+
+The image metadata will be transformed to match the signature of imported images, and therefore can be passed directly to the `<Image />` component or the `getImage()` function.
+
+```ts
+import { image, defineCollection, z } from "astro:content";
+
+const blogCollection = defineCollection({
+  schema: z.object({
+    title: z.string(),
+    cover: image().refine((img) => img.width >= 1080, {message: 'Cover image must be at least 1080 pixels wide!'}),
+  }),
+});
+```
+
+## `astro:assets` module
+
+Enabling asset support allows you to access the `astro:assets` module. This module exposes the following features:
+
+- `Image`
+- `getImage()`
+
+### `<Image />` (`astro:assets`)
+
+The `<Image />` component generates an `<img>` tag.
+
+This component can transform an image's dimensions, file type, and quality to produce an optimized image that avoids Content Layout Shift **(CLS)**.
```

Comment:
Technically, it's not the generated image that avoids CLS, it's the `img` tag generated by the `Image` component.